[qt5-base] build failure

**Host Environment**
- OS: Windows WSL ubuntu 20.04
- Compiler: g++ (10)

**To Reproduce**
vcpkg install qt5-charts

**Failure logs** 
```
Starting package 2/8: qt5-base:x64-linux
Building package qt5-base[core]:x64-linux...
-- Figuring out qt target mkspec. Target arch x64
-- Target mkspec set to: linux-g++
-- Host mkspec not set. Qt's own buildsystem will try to figure out the host system
-- Downloading http://download.qt.io/official_releases/qt/5.12/5.12.8/submodules/qtbase-everywhere-src-5.12.8.tar.xz...
-- Extracting source /home/john/.vcpkg/downloads/qtbase-everywhere-src-5.12.8.tar.xz
-- Applying patch patches/winmain_pro.patch
-- Applying patch patches/windows_prf.patch
-- Applying patch patches/qt_app.patch
-- Applying patch patches/gui_configure.patch
-- Applying patch patches/icu.patch
-- Applying patch patches/xlib.patch
-- Applying patch patches/Qt5BasicConfig.patch
-- Applying patch patches/Qt5PluginTarget.patch
-- Applying patch patches/prl_parser.patch
-- Applying patch patches/Qt5GuiConfigExtras.patch
-- Using source at /home/john/.vcpkg/buildtrees/qt5-base/src/5.12.8-58a5dc6709
-- Configuring x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: /home/john/.vcpkg/buildtrees/qt5-base/src/5.12.8-58a5dc6709/configure -confirm-license -opensource -verbose -system-zlib -system-libjpeg -system-libpng -system-freetype -system-pcre -system-doubleconversion -system-sqlite -system-harfbuzz -icu -no-angle -openssl-linked -fontconfig -xcb-xlib -linuxfb -nomake examples -nomake tests -static LIBJPEG_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libjpeg.a ZLIB_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libz.a "LIBPNG_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libpng16d.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libz.a" PCRE2_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libpcre2-16.a "FREETYPE_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libfreetyped.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libbz2d.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libpng16d.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libz.a" "ICU_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libicui18n.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libicutu.a  /home/john/.vcpkg/installed/x64-linux/debug/lib/libicuuc.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libicuio.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libicudata.a" QMAKE_LIBS_PRIVATE+=/home/john/.vcpkg/installed/x64-linux/debug/lib/libbz2d.a QMAKE_LIBS_PRIVATE+=/home/john/.vcpkg/installed/x64-linux/debug/lib/libpng16d.a "PSQL_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libpq.so /home/john/.vcpkg/installed/x64-linux/debug/lib/libssl.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libcrypto.a -ldl -lpthread" "SQLITE_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libsqlite3.a -ldl -lpthread" HARFBUZZ_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libharfbuzz.a "OPENSSL_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libssl.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libcrypto.a -ldl -lpthread" "FONTCONFIG_LIBS=/home/john/.vcpkg/installed/x64-linux/debug/lib/libfontconfig.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libfreetyped.a /home/john/.vcpkg/installed/x64-linux/debug/lib/libexpat.a" -debug -prefix /home/john/.vcpkg/installed/x64-linux -extprefix /home/john/.vcpkg/installed/x64-linux -hostprefix /home/john/.vcpkg/installed/x64-linux/tools/qt5/debug -hostlibdir /home/john/.vcpkg/installed/x64-linux/tools/qt5/debug/lib -hostbindir /home/john/.vcpkg/installed/x64-linux/tools/qt5/debug/bin -archdatadir /home/john/.vcpkg/installed/x64-linux/tools/qt5/debug -datadir /home/john/.vcpkg/installed/x64-linux/share/qt5/debug -plugindir /home/john/.vcpkg/installed/x64-linux/debug/plugins -qmldir /home/john/.vcpkg/installed/x64-linux/debug/qml -headerdir /home/john/.vcpkg/installed/x64-linux/include -libexecdir /home/john/.vcpkg/installed/x64-linux/tools/qt5/debug -bindir /home/john/.vcpkg/installed/x64-linux/debug/bin -libdir /home/john/.vcpkg/installed/x64-linux/debug/lib -I /home/john/.vcpkg/installed/x64-linux/include -L /home/john/.vcpkg/installed/x64-linux/debug/lib -L /home/john/.vcpkg/installed/x64-linux/debug/lib/manual-link -xplatform linux-g++
    Working Directory: /home/john/.vcpkg/buildtrees/qt5-base/x64-linux-dbg
    Error code: 3
    See logs for more information:
      /home/john/.vcpkg/buildtrees/qt5-base/config-x64-linux-dbg-out.log
      /home/john/.vcpkg/buildtrees/qt5-base/config-x64-linux-dbg-err.log

Call Stack (most recent call first):
  ports/qt5-base/cmake/configure_qt.cmake:110 (vcpkg_execute_required_process)
  ports/qt5-base/portfile.cmake:272 (configure_qt)
  scripts/ports.cmake:90 (include)


Error: Building package qt5-base:x64-linux failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: qt5-base:x64-linux
  Vcpkg version: 2020.02.04-unknownhash

Additionally, attach any relevant sections from the log files above.
```

--- a/ports/qt5-base/CONTROL
+++ b/ports/qt5-base/CONTROL
@@ -1,5 +1,5 @@
 Source: qt5-base
-Version: 5.12.8-2
+Version: 5.12.8-3
 Homepage: https://www.qt.io/
 Description: Qt5 Application Framework Base Module. Includes Core, GUI, Widgets, Networking, SQL, Concurrent and other essential qt components.
 Build-Depends: zlib, libjpeg-turbo, libpng, freetype, pcre2, harfbuzz, sqlite3, libpq, double-conversion, openssl, angle (!windows), egl-registry, icu (!uwp), fontconfig (!windows)

--- a/ports/qt5-base/portfile.cmake
+++ b/ports/qt5-base/portfile.cmake
@@ -6,6 +6,10 @@ else()
     option(QT_OPENSSL_LINK "Link against OpenSSL at compile-time." OFF)
 endif()
 
+if (VCPKG_TARGET_IS_LINUX)
+    message(WARNING "${PORT} currently requires the following libraries from the system package manager:\n    libx11-xcb-dev\n\nThese can be installed on Ubuntu systems via apt-get install libx11-xcb-dev.")
+endif()
+
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
 

